nix/overlays/wire-server.nix

```diff
@@ -14,7 +14,13 @@ self: super: {
           sha256 = "1i9dlhw0xk1viglyhail9fb36v1awrypps8jmhrkz8k1bhx98ci3";
         };
         cargoSha256 = "0zs8ibv7rinrrzp9naxd7yak7kn1gp3pjb3g8i4wf7xw2hkkq81z";
+
+        patchLibs = super.lib.optionalString super.stdenv.isDarwin ''
+            install_name_tool -id $out/lib/libcryptobox.dylib $out/lib/libcryptobox.dylib
```

Comment:
and this was absolutely needed? I thought it fixed itself by just bumping `nixpkgs` no?

The fact that `buildRustPackage` outputs binaries with broken paths sounds like something we should report as a bug upstream.

Reply:
That was PBUK. Paths looked ok.... but not when I scrolled up. And it didn't build until I made that change.

It's a known issue apparently https://discourse.nixos.org/t/patchelf-for-mach-o-shared-libraries/562/6 there's even a hook for it `fixDarwinDylibNames`.